tensorboard/components/tf_tensorboard/autoReloadBehavior.ts

```diff
@@ -39,13 +39,28 @@ namespace tf_tensorboard {
       _autoReloadId: {
         type: Number,
       },
+      // Tracks whethere an auto reload was missed because the document was not visible.
+      _missedAutoReload: {
+        type: Boolean,
+        value: false,
+      },
       autoReloadIntervalSecs: {
         type: Number,
         value: 30,
       },
     },
+    attached: function() {
+      document.addEventListener(
+        'visibilitychange',
+        this._handleVisibilityChange.bind(this)
+      );
+    },
     detached: function() {
       window.clearTimeout(this._autoReloadId);
+      document.removeEventListener(
+        'visibilitychange',
+        this._handleVisibilityChange.bind(this)
```

Comment:
I believe `bind` call creates a new reference and this is not removing the event listener attached at L55.

Reply:
Ya, yuck. I looked at how some other components handle this and picked what looks like the least-bad option.